### PR TITLE
doc_source: highlight kube-proxy upgrades for EKS 1.16

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

File: doc_source/update-cluster.md [hunk 1]
```diff
@@ -300,3 +303,151 @@ The previous command may use different default values from what is set in your c
...
+                operator: NotIn
+                values:
+                  - fargate
+      hostNetwork: true
+      tolerations:
+      - operator: "Exists"
+      priorityClassName: system-node-critical
+      containers:
+      - name: kube-proxy
+        image: 602401143452.dkr.ecr.REGION.amazonaws.com/eks/kube-proxy:v1.16.8
```
Comment (mogren) [inline doc_source/update-cluster.md:408]: Nit: Maybe we should explain that `REGION` has to be replaced. Also, do we have a list of the non-standard ECR repos that are not `602401143452`, like BAH, HKG, BJS and ZHY?

Comment (gyuho) [reply doc_source/update-cluster.md:408]: ^ @HLeeHart

Comment (gyuho) [reply doc_source/update-cluster.md:408]: How about

```
image: ACCOUNT-ID.dkr.ecr.REGION.amazonaws.com/eks/kube-proxy:v1.16.8
# ACCOUNT-ID is the same account ID from the existing kube-proxy daemon set spec
```

**Comment by jontro** [comment]
When I make these changes I get the following error:
 1 server.go:443] invalid configuration: no server found for cluster "default"

How should the configmap named kube-proxy look like? If I list my current one it looks like the following
```

apiVersion: v1
data:
  kubeconfig: |-
    kind: Config
    apiVersion: v1
    clusters:
    - cluster:
        certificate-authority: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
      name: default
    contexts:
    - context:
        cluster: default
        namespace: default
        user: default
      name: default
    current-context: default
    users:
    - name: default
      user:
        tokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
kind: ConfigMap
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"v1","data":{"kubeconfig":"kind: Config\napiVersion: v1\nclusters:\n- cluster:\n    certificate-authority: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt\n  name: default\ncontexts:\n- context:\n    cluster: default\n    namespace: default\n    user: default\n  name: default\ncurrent-context: default\nusers:\n- name: default\n  user:\n    tokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token"},"kind":"ConfigMap","metadata":{"annotations":{},"labels":{"eks.amazonaws.com/component":"kube-proxy","k8s-app":"kube-proxy"},"name":"kube-proxy","namespace":"kube-system"}}
  creationTimestamp: "2019-05-03T15:42:01Z"
  labels:
    eks.amazonaws.com/component: kube-proxy
    k8s-app: kube-proxy
  name: kube-proxy
  namespace: kube-system
  resourceVersion: "36"
  selfLink: /api/v1/namespaces/kube-system/configmaps/kube-proxy
```

**Comment by jontro** [comment]
Also this pull request seems to have been reverted in periodic updates

**Comment by nithu0115** [comment]
#131

**Pull request merged**